os/arch/Kconfig.board

```diff
@@ -16,6 +16,7 @@ config ARCH_BOARD_ARTIK053
 	bool "Samsung ARTIK-053 Starter Kit"
 	depends on ARCH_CHIP_S5JT200
 	select SAMSUNG_NS2
+	select ARCH_HAVE_PM
```

Comment:
This is same as 053s and 055s as well. Let's move this under ARCH_BOARD_ARTIK05X_FAMILY.